packages/api-client/tests/test-granules.js

```diff
@@ -44,7 +46,7 @@ test('getGranule calls the callback with the expected status codes', async (t) =
     payload: {
       httpMethod: 'GET',
       resource: '/{proxy+}',
-      path: `/granules/${t.context.granuleId}`,
+      path: `/granules/${t.context.collectionId}/${t.context.granuleId}`,
```

Comment:
I think some of these test changes were inadvertant as the intention of this test was to test the `API_URI/{granuleId}` path, whereas the test for including collectionId is at L106